tests/unit/select_connection_ioloop_tests.py

```diff
@@ -81,7 +89,54 @@ def start_test(self):
         self.start()
 
     def on_timer(self, val):
-        self.assertEquals(val, self.timer_stack.pop())
+        self.assertEqual(val, self.timer_stack.pop())
+        if not self.timer_stack:
+            self.ioloop.stop()
+
+    def test_normal(self):
+        self.start_test()
+
+    def test_timer_for_suicide(self):
```

Comment:
Need a short docstring that describes what this is testing. "test_timer_for_suicide" is not meaningful to other people.